### DISCUSSION TITLE
Why is adns still listed as a dependency for GnuPG when it has been deprecated and removed since GPG2.1?

### DISCUSSION THREAD

**Discussion by cupcake99** [Homebrew/discussions#1227] [Everyday usage]
Hi, i was trying to install GPG today and ran into a build failure with adns which is listed as a dependency on the formula (known issue with my old version of OSX, no native time.h, my problem to deal with). But, as it turns out this library has been removed from GPG since v.2.1, as referenced here https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=commit;h=2e734a3ce159de8fb60df2bd5d454f98ca710717

I removed the dep's line from my gnupg.rb formula and the install went ahead with zero issues. It's obviously not a bug, but is it worth making a pull request? I just wanted to point this out to see if anybody knows a reason why it hasn't been removed yet. Thanks.

**Comment by carlocab** [comment]
We probably just missed the memo :)

Feel free to open a PR.

**Reply by cupcake99** [reply]
Thanks, just submitted the PR.
